### PR TITLE
Exp pe config

### PR DESCRIPTION
**Pull request by bobcheng15**
1. added configuration support for instructions fp_add, fp_mul, fp_max, and, fgetfint, fgetffrac, faddiexp
2. added unittest to test the alu under fp_max, fgetfint, fgetffrac, faddiexp configurations
3. enabled constant operand  feature for fp_mul and bitwise and  configurations for the alu
4. fix the imporlib_resouces bug in the CI check

### CODE REVIEW

File: lake/modules/onyx_pe.py [hunk 1]
```diff
@@ -412,13 +397,26 @@ def get_bitstream(self, config_kwargs):
...
-            # support constant operand for and and fp_mul for now
-            assert op == 5 or op == 6
-            # only accepting one input from port a【
-            sparse_num_inputs = 0b001
+        sparse_num_inputs = 0b000
+        if 'use_dense' in config_kwargs and config_kwargs['use_dense'] is True:
+            override_dense = True
+            config += [("dense_mode", 1)]
+        elif 'pe_connected_to_reduce' in config_kwargs and config_kwargs['pe_connected_to_reduce'] is True:
+            # If this flag is set, we want to connect the pe to reduce
```
Comment (steveri) [inline lake/modules/onyx_pe.py]: > # If this flag is set, we want to connect the pe to reduce

Thanks for adding comments! But this comment needs a little more thought maybe.
"to reduce"...reduce what? code size? by...? bypassing the fifos?

Comment (kalhankoul96) [reply lake/modules/onyx_pe.py]: reducer is a sparse primitive, @bobcheng15  can include that in the comments

Comment (bobcheng15) [reply lake/modules/onyx_pe.py]: @steveri sorry for the confusion. Like Kalhan says, by setting this `pe_connected_to_reduce` flag, the ALU (or onyx_pe) is connected to the reduce sparse primitive. I will revise the comment to make it clear.

**Review by kalhankoul96 — APPROVED** [review]
LGTM

**Pull request merged**